packages/core/src/components/meter/meter.spec.ts

```diff
@@ -0,0 +1,106 @@
+        type="neutral"
+        size="large"
+        title=":fialho-dino:"
+        min=1
+        max=6
+        actual=4
+      ></atom-meter>`,
+    })
+
+    await page.waitForChanges()
+
+    expect(page.root).toEqualHtml(`
+      <atom-meter type="neutral" size="large" actual="4" max="6" min="1" title=":fialho-dino:">
+        <mock:shadow-root>
+          <div class="container-text ">
+            <h1 class="title">
+              :fialho-dino:
+            </h1>
+            <slot></slot>
+          </div>
+          <div class="atom-meter is-large">
+            <div class="within is-neutral" style="width: 60%;"></div>
+          </div>
+        </mock:shadow-root>
+      </atom-meter>
+    `)
+  })
+
+  it('should render with centerTitle prop', async () => {
+    const page = await newSpecPage({
+      components: [AtomMeter],
+      html: `<atom-meter center-title="true" title="Centered Title" min="1" max="6" actual="4"></atom-meter>`,
+    })
+
+    await page.waitForChanges()
+
+    expect(page.root).toEqualHtml(`
+      <atom-meter center-title="true" title="Centered Title" min="1" max="6" actual="4">
+        <mock:shadow-root>
+          <div class="container-text hasCenterTitle">
+            <h1 class="title">Centered Title</h1>
+          </div>
+          <div class="atom-meter is-large">
+            <div class="within is-neutral" style="width: 60%;"></div>
+          </div>
+        </mock:shadow-root>
+      </atom-meter>
+    `)
+  })
+
+  it('should render with size small prop', async () => {
+    const page = await newSpecPage({
+      components: [AtomMeter],
+      html: `<atom-meter title=":wessel-quase-coringa:" min="1" max="6" actual="4" size="small"></atom-meter>`,
+    })
+
+    await page.waitForChanges()
+
+    expect(page.root).toEqualHtml(`
+      <atom-meter title=":wessel-quase-coringa:" min="1" max="6" actual="4" size="small">
+        <mock:shadow-root>
+          <div class="container-text">
+            <h1 class="title">:wessel-quase-coringa:</h1>
+            <slot></slot>
+          </div>
+          <div class="atom-meter is-small">
+            <div class="within is-neutral" style="width: 60%;"></div>
+          </div>
+        </mock:shadow-root>
+      </atom-meter>
+    `)
+  })
+
+  it('should render with children', async () => {
+    const page = await newSpecPage({
+      components: [AtomMeter],
+      html: `<atom-meter title="test" min="1" max="6" actual="4">Children</atom-meter>`,
+    })
+
+    await page.waitForChanges()
+
+    expect(page.root).toEqualHtml(`
+      <atom-meter title="test" min="1" max="6" actual="4">
+        <mock:shadow-root>
+          <div class="container-text">
+            <h1 class="title">test</h1>
+            <slot></slot>
+          </div>
+          <div class="atom-meter is-large">
+            <div class="within is-neutral" style="width: 60%;"></div>
+          </div>
+        </mock:shadow-root>
+        Children
+      </atom-meter>
+    `)
+  })
+})
```

Comment:
guess it is missing the mobile scenario

Reply:
We will no longer be able to test, because I migrated to breakpoints instead of handling the class through javascript

Commit: https://github.com/juntossomosmais/atomium/commit/ce3e6d71f8999342744e3f3d0f97778ec5b7e8ec

Reply:
Ooh, nice!!

Made: https://github.com/juntossomosmais/atomium/pull/695/commits/1465f8ac06ba2feaf3a56cdcda7ec636b3f493c7